src/store/reset-password/saga.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expectSaga } from 'redux-saga-test-plan';
+import { call } from 'redux-saga/effects';
+import { resetPasswordPage } from './saga';
+import { resetPassword as resetPasswordApi } from './api';
+import { setLoading, setErrors, setEmailSubmitted, SagaActionTypes } from './index';
+import { ResetPasswordErrors } from '.';
+
+describe('resetPasswordPage', () => {
```

Comment:
I have some notes about these tests but I think I'm going to record a video to try to explain/show what I mean.